assets/src/dashboard/components/pageStructure/menuButton.js

```diff
@@ -41,12 +42,10 @@ const TransparentButton = styled.button`
   height: 24px;
   padding: 0;
   margin-right: 16px;
-  border: ${({ theme }) => theme.DEPRECATED_THEME.borders.transparent};
   background: transparent;
-
-  &:focus {
-    border: ${({ theme }) => theme.DEPRECATED_THEME.borders.action};
-  }
+  border: 0;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  ${themeHelpers.focusableOutlineCSS}
```

Comment:
small nit: may not need the `;`
```suggestion
  ${themeHelpers.focusableOutlineCSS};
```

Reply:
It breaks when I add the semicolon 🤷 the return doesn't look like it should have a semi to me:
```
export const focusableOutlineCSS = (colorOrProps, background) => {
  const accent =
    typeof colorOrProps === 'string'
      ? colorOrProps
      : colorOrProps?.theme?.colors?.border?.focus;
  return css`
    &.${FOCUS_VISIBLE_SELECTOR}, &[data-focus-visible-added] {
      ${focusCSS(accent, background)};
    }
  `;
};
```